modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -191,6 +191,20 @@ def build_target_veteran(veteran_id:, loa:) # rubocop:disable Metrics/MethodLeng
         target_veteran[:va_profile] = ClaimsApi::Veteran.build_profile(mpi_profile.birth_date)
         target_veteran
       end
+
+      def file_number_check
+        if !target_veteran.mpi.birls_id.empty?
```

Comment:
why are we doing `!` of `empty?` and not doing something like `present?`? Also, assuming because we've built target veteran we shouldn't see the `target_veteran.mpi` come back as nil, but do we have something to handle if we do?